docs/source/getting-started.md

```diff
@@ -0,0 +1,21 @@
+# Getting started
+
+`menuinst` can be used to create shortcuts or menu items across operating systems.
+It's designed to integrate nicely with `conda` packages, 
+but it can be used without it to an extent.
+
+## Installation
+
+`menuinst` is usually installed along with `conda` in the `base` environment.
+So, in most cases, you won't have to do much. For other environments, you can do:
+
+```console
+$ conda install "menuinst>=2"
```

Comment:
`menuinst` will always be available from default channels ?

Reply:
It should be, yes.